pkg/skaffold/debug/transform_python.go

```diff
@@ -139,14 +162,46 @@ func extractPtvsdArg(args []string) *ptvsdSpec {
 	return &spec
 }
 
-func hasPtvsdModule(args []string) bool {
-	if index := util.StrSliceIndex(args, "-mptvsd"); index >= 0 {
+func extractDebugpySpec(args []string) *pythonSpec {
+	if !hasPyModule("debugpy", args) {
+		return nil
+	}
+	spec := pythonSpec{debugger: debugpy, port: -1}
+	for i, arg := range args {
+		switch arg {
+		case "--listen":
+			if i == len(args)-1 {
+				return nil
+			}
+			s := strings.SplitN(args[i+1], ":", 2)
+			if len(s) > 1 {
+				spec.host = s[0]
+			}
+			port, err := strconv.ParseInt(s[len(s)-1], 10, 32)
+			if err != nil {
+				logrus.Errorf("Invalid port %q: %s\n", args[i+1], err)
+				return nil
+			}
+			spec.port = int32(port)
+
+		case "--wait-for-client":
+			spec.wait = true
+		}
+	}
+	if spec.port < 0 {
+		return nil
+	}
+	return &spec
+}
+
+func hasPyModule(module string, args []string) bool {
+	if index := util.StrSliceIndex(args, "-m"+module); index >= 0 {
```

Comment:
nit for consistency:
```suggestion
	if index := util.StrSliceIndex(args, fmt.Sprintf("-m%s", module)); index >= 0 {
```